docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -31,6 +44,28 @@ To follow this guide, you need:
 - Docker
 - The docker-compose plugin
 
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Quick start for Tempo](https://killercoda.com/grafana-labs/course/tempo/quick-start).
+
+It's a fully configured environment with all the dependencies already installed.
+
+![Interactive](/media/docs/tempo/tempo-ile.svg)
```

Comment:
This large banner is out of place in the docs. The admonition already highlights the tutorial. Isn't the text in the green highlight enough? 
```suggestion
```
This is what it looks like: 
![image](https://github.com/user-attachments/assets/ecc68bed-7dfc-45be-89ac-147909e3dc7c)